docker/ci/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM pcic/pdp-base-minimal-unsafe:1.0.0
+FROM pcic/pdp-base-minimal-unsafe:pdp-python3
```

Comment:
what does "unsafe" mean here?

Reply:
See the [pdp-docker](https://github.com/pacificclimate/pdp-docker) project for documentation on that